tests/ignite/metrics/test_frequency.py

```diff
@@ -35,9 +35,10 @@ def update_fn(engine, batch):
 
     engine = Engine(update_fn)
     wps_metric = Frequency(output_transform=lambda x: x["ntokens"], device=device)
-    wps_metric.attach(engine, "wps")
+    event = Events.ITERATION_COMPLETED if every == 1 else Events.ITERATION_COMPLETED(every=every)
```

Comment:
Maybe we can just use `Events.ITERATION_COMPLETED(every=every)` ? It works now with `every=1`.

Reply:
Is it the case that `Events.ITERATION_COMPLETED(every=1)` is equivalent to `Events.ITERATION_COMPLETED`? For some reason I thought there was some nuanced difference... maybe not, though.

Reply:
Ah - just checked the `CallableEventWithFilter.__call__`; it seems like they're forced to equivalent explicitly. Will update the test.

Reply:
Now `Events.ITERATION_COMPLETED(every=1) == Events.ITERATION_COMPLETED`.
https://github.com/pytorch/ignite/blob/master/ignite/engine/events.py#L78